yesod-auth-oauth2.cabal

```diff
@@ -1,5 +1,5 @@
 name:            yesod-auth-oauth2
-version:         0.1.4
+version:         0.1.5
```

Comment:
Can you back out the version bump? I can take care of that separately after we merge.